docs/user_guide/make_grid.md

```diff
@@ -17,13 +17,14 @@ combine_grid -vb *.grd > [outputfile].grd
 
 The `make_grid` routine provides a lot of options for customizing the gridding process, such as the altitude at which the mapping is performed, or whether to exclude ground/ionospheric scatter. The optional `-xtd` ("extended") flag will include the power and spectral width parameters in the output file, in addition to the velocity. Type `make_grid --help` for more information. 
 	
-## Full-day grid files
-The standard practice is to generate 24-hour grid files. If you have already made a concatenated 24-hour fitacf file, then use that file in the manner shown above. Otherwise, you can use the `-c` flag to get RST to concatenate the input fitacf files on-the-fly. This works only when the input files are all from the same radar; the method for combining data from multiple radars into a single grid file is shown in the next section.
+	
+### Full-day grid files
+The standard practice is to generate 24-hour grid files. If you have already made a concatenated 24-hour fitacf file, then use that file in the manner shown above. Alternatively, you can provide several fitacf files as input to `make_grid`, and the routine will automatically concatenate the files for you. This works only when the input files are all from the same radar; the method for combining data from multiple radars into a single grid file is shown in the next section.
```

Comment:
Didn't we find a bug with 24-hour fitacf file in make_grid where it didn't read in the record right if it jumped too much.

Reply:
@mts299 I am not aware of this bug. Do you have an example?

Reply:
Actually you were the one that found it way back when... I can try but it may take me a day or two.

Reply:
@mts299 This PR is about updating the documentation to reflect the changes made to the `make_grid` command line options in #373. I suggest we open a new issue if there's a bug in `make_grid`

Reply:
Correct, I am just saying we may want to take the line out about using 24-hour fitacf files.